central/src/main/java/org/glowroot/central/util/ClusterManager.java

```diff
@@ -116,14 +116,20 @@ public static ClusterManager create(File confDir, Map<String, String> jgroupsPro
 
         private ClusterManagerImpl(File confDir, String jgroupsConfigurationFile,
                 Map<String, String> jgroupsProperties) {
-            GlobalConfiguration configuration = new GlobalConfigurationBuilder()
-                    .transport()
-                    .defaultTransport()
-                    .addProperty("configurationFile",
-                            getConfigurationFilePropertyValue(confDir, jgroupsConfigurationFile))
-                    .globalJmxStatistics()
-                    .enable()
-                    .build();
+            GlobalConfigurationBuilder builder = new GlobalConfigurationBuilder();
+            builder
+                .transport()
+                .defaultTransport()
+                .addProperty("configurationFile",
+                        getConfigurationFilePropertyValue(confDir, jgroupsConfigurationFile))
+                .jmx()
+                .enable()
+                .serialization()
+                    .marshaller(new JavaSerializationMarshaller())
+                        .allowList()
+                            .addRegexp("org\\.glowroot\\.central\\.repo\\.Immutable.*");
```

Comment:
This should allow java serialization again for all classes matching this pattern